e2e/atlas_clusters_test.go

```diff
@@ -49,27 +49,35 @@ func TestAtlasClusters(t *testing.T) {
 		cmd := exec.Command(cliPath,
 			atlasEntity,
 			clustersEntity,
-			"create", clusterName,
+			"create",
+			clusterName,
 			"--region=US_EAST_1",
 			"--members=3",
-			"--instanceSize=M5",
+			"--instanceSize=M10",
 			"--provider=AWS",
-			"--mdbVersion=4.2")
+			"--mdbVersion=4.0",
+			"--diskSizeGB=10")
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
 
 		if err != nil {
 			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
 		}
 
-		cluster := mongodbatlas.Cluster{}
-		err = json.Unmarshal(resp, &cluster)
+		cluster := new(mongodbatlas.Cluster)
+		err = json.Unmarshal(resp, cluster)
 		if err != nil {
 			t.Fatalf("unexpected error: %v", err)
 		}
 
 		if cluster.Name != clusterName {
-			t.Errorf("got=%#v\nwant=%#v\n", cluster.Name, clusterName)
+			t.Errorf("Name, got=%#v\nwant=%#v\n", cluster.Name, clusterName)
+		}
+		if cluster.MongoDBMajorVersion != "4.0" {
+			t.Errorf("MongoDBMajorVersion, got=%#v\nwant=%#v\n", cluster.MongoDBMajorVersion, "4.0")
+		}
+		if *cluster.DiskSizeGB != 10 {
+			t.Errorf("MongoDBMajorVersion, got=%#v\nwant=%#v\n", cluster.DiskSizeGB, 10)
```

Comment:
[np] define this in a reused const of some form - it's used here and above on the Run command, so would be better to reuse a single param.